tools/run_local_ironic.sh

```diff
@@ -180,12 +188,15 @@ if [[ "${CONTAINER_RUNTIME}" == "podman" ]]; then
 fi
 
 # Start image downloader container
-# shellcheck disable=SC2086
-sudo "${CONTAINER_RUNTIME}" run -d --net host --privileged --name ipa-downloader \
-     ${POD} --env-file "${IRONIC_DATA_DIR}/ironic-vars.env" \
-     -v "$IRONIC_DATA_DIR:/shared" "${IPA_DOWNLOADER_IMAGE}" /usr/local/bin/get-resource.sh
+IPA_DOWNLOAD_ENABLED="${IPA_DOWNLOAD_ENABLED:-true}"
```

Comment:
The default value should be set before it's used on line 161

Reply:
Fixed.